src/main/webapp/css/style.css

```diff
@@ -201,4 +201,24 @@ body.fade-out {
 #user-name{
   padding: 16px;
   font-size: 16px;
-}
\ No newline at end of file
+}
+
+/* ==========================================================================
+   Maps
+   ========================================================================== */
+   .map-container{
```

Comment:
fix indentation and add space before brace

ditto through to the end of this file